static/templates/settings/bot-settings.handlebars

```diff
@@ -84,11 +84,11 @@
     <div id="get_api_key_box" class="side-padded-container">
       {{#tr this}}
       <p>Please re-enter your password to confirm your identity.
-        (<a href="/accounts/password/reset/" target="_blank">Forgotten it?</a>)</p>
+        <a href="/accounts/password/reset/" target="_blank">{{t "Forgotten it?" }}</a></p>
       {{/tr}}
       <form action="/json/fetch_api_key" method="post" class="form-horizontal">
         <div class="control-group">
-          <label for="password" class="control-label">{{t "Current password" }}</label>
+          <label for="password" class="control-label">{{t "Current password" }}</label>&nbsp;
```

Comment:
Right, so the way I'd do this is put it in a separate commit in the same PR.  The worst option is to do nothing when you notice it -- then we won't fix it!  A close second is to have it be hidden into a commit that's changing something else, since that's either going to result in the change being unreviewed entirely (since there's a chance the reviewer won't even notice it) or require an extra round-trip of disucssion to understand the purpose of the change as we're having now :).  

Since commits are essentially free, it definitely belongs in its own commit, and that commit's commit message can explain the purpose of the change to make it easy for a reviewer to check your work.  

As to the "different PR" question, while it has nothing to do with i18n, it's a minor tweak that touches the same code as the other changes in the PR, and thus is reasonable to submit at the same time.  PRs are also basically free; the question is really about whether you save more time by doing the reviews together or separately.  Tacking a minor fix as a bonus commit on another PR is usually pretty reasonable.

Does that make sense?  I feel like our documentation on commit structure could still use some work to explain this stuff super crisply :/.